app/recipe/tests.py

```diff
@@ -0,0 +1,109 @@
+import uuid
+from django.urls import reverse
+from django.test import TestCase
+
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from .models import Recipe, Ingredient
+from .serializers import RecipeSerializer
+
+RECIPE_URL = reverse('recipe:recipe-list')
+
+def recipe_detail_url(recipe_id):
+    return reverse('recipe:recipe-detail', args=[recipe_id])
+
+
+class RecipeApiTest(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_get_all_recipes(self):
+        Recipe.objects.create(name='Tortilla', description='Tortilla de patatas con cebolla')
+        Recipe.objects.create(name='Banderilla', description='Pincho de olivas y boquerones')
+
+        recipes = Recipe.objects.all()
+        serializer = RecipeSerializer(recipes, many=True)
+        res = self.client.get(RECIPE_URL)
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        self.assertEqual(res.data, serializer.data)
```

Comment:
It would be helpful to assert `serializer.data` against a literal dictionary of values, so the reader of the test case can see the structure of the data, and also be assured that e.g. both the Serializer and the endpoint are actually returning something.

Reply:
is there any way to do this assertion without parsing both structures? If I just try to compare them I get an ordered dictionary with ID's from the serializer that doesn´t match a simple dictionary